accepted/PSR-17-http-factory.md

```diff
@@ -117,6 +123,9 @@ interface StreamFactoryInterface
      * Create a new stream from a string.
      *
      * The stream SHOULD be created with a temporary resource.
+     * 
+     * The stream MUST be created with the current position of the file read/write
+     * pointer at the beginning of the stream. (Errata 9.1)
```

Comment:
@mindplay-dk on closer inspection, this one is not correct. Given that this is an "open + write" operation, it would be natural for the pointer to be at the end of the stream and there is no guarantee that the stream can be rewound, making it impossible to place the pointer at the beginning of the stream.

Instead, I believe this can be combined with the previous line and read:

> The stream SHOULD be created with a temporary resource, and the file read/write pointer SHOULD be placed consistently with an `fwrite()` operation.